sdk/search/ci.yml

```diff
@@ -4,7 +4,7 @@
 resources:
   repositories:
     - repository: azure-sdk-build-tools
-      type: github
+      type: git
```

Comment:
Out of curiosity how does this error manifest itself? Does it simply fail to find this template?